### PR TITLE
Add changelog and fix readme

### PR DESCRIPTION
**Pull request by SamirBoulil**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,14 +38,14 @@ Please note that you will certainly need to provide your GitHub credentials with
 A lot of our dependencies are coming from GitHub and this reaches the max limit of 50 API calls from anonymous users.
 
 ```
-    $ php composer.phar create-project --prefer-dist akeneo/pim-community-standard ./pim-project "2.1.*@stable"
+    $ php composer.phar create-project --prefer-dist akeneo/pim-community-standard ./pim-project "2.2.*@stable"
```
Comment (damien-carcel) [inline README.md]: 2.3-dev@dev ?

File: README.md [hunk 2]
```diff
@@ -38,14 +38,14 @@ Please note that you will certainly need to provide your GitHub credentials with
...
 ```
 
-After that, follow the instructions here: https://docs.akeneo.com/2.1/install_pim/installation_ce_archive.html
+After that, follow the instructions here: https://docs.akeneo.com/2.2/install_pim/installation_ce_archive.html
```
Comment (damien-carcel) [inline README.md]: master rather than 2.2 ?

File: README.md [hunk 3]
```diff
@@ -38,14 +38,14 @@ Please note that you will certainly need to provide your GitHub credentials with
...
 
 Upgrade instructions
 --------------------
 
 To upgrade Akeneo PIM to a newer version, please follow:
-https://docs.akeneo.com/2.1/migrate_pim/index.html
+https://docs.akeneo.com/2.2/migrate_pim/index.html
```
Comment (damien-carcel) [inline README.md]: same

File: README.md (hunk 1 above)
Comment (SamirBoulil) [reply README.md]: Normally should be done in the 2.3 release no ? But ok I'll do it

Comment (damien-carcel) [reply README.md]: As the master branch has the alias 2.3, I think 2.3-dev@dev can work. It will have to be changed for 2.3.*@stable when 2.3 is released.

**Pull request merged**